components/member-profile/index.js

```diff
@@ -1,14 +1,14 @@
-import React, { useState, useEffect } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import SocialMediaIcon from '../social-media-icon';
-import { getPRsbyUser } from '../../helper-functions/getPRs';
 import getBadges from './mock/get-badges';
 import classNames from './member-profile.module.scss';
 
 const Profile = (props) => {
   const {
     membersData: { id, first_name, last_name, company, designation },
-    imageLink
+    imageLink,
+    pullRequests
   } = props;
   const { membersData } = props;
```

Comment:
Could you please remove this line of code, I think this is not needed since we are destructuring on line no 9.

Reply:
Sorry My bad, this line of code is also required. On line no 9 we are not completely destruturing the object. There are social media links also included in this `membersData`
I will improve this code in my refactor task.